src/components/Todolist.tsx

```diff
@@ -1,10 +1,86 @@
-import React from 'react';
-
-export const Todolist = () => {
-    return (
-        <div>
-
-        </div>
-    );
-};
-
+import React, { ChangeEvent, KeyboardEvent, useState } from 'react';
+import { FilterType, TasksType } from '../App';
+import './Todolist.css';
+
+export type TodolistType = {
+  title: string;
+  tasks: TasksType[];
+  removeTask: (id: string) => void;
+  filterTask: (filter: FilterType) => void;
+  addTask: (title: string) => void;
+  changeCheckStatus: (isDone: boolean, id: string) => void;
+  filter: FilterType;
+};
+export const Todolist = (props: TodolistType) => {
```

Comment:
Каждое слово в `CamelCase` принято начинать с большой буквы) 
Ещё типы для свойств компонентов принято называть `<имя компонента>Props`:

```suggestion
export type TodoListProps = {
  title: string;
  tasks: Task[]; 
  removeTask: (id: string) => void;
  filterTask: (filter: FilterType) => void;
  addTask: (title: string) => void;
  changeTaskStatus: (isDone: boolean, id: string) => void;
  filter: TaskFilter;
};

export const TodoList = (props: TodoListProps) => {
```